tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Calculate facts
+  set_fact:
+    build_base: "{{ (mumble_web.build_base | default(mumble_web_build_base)) | regex_replace('\\/$', '') }}"
+    deploy_path: "{{ (mumble_web.deploy_path | default(mumble_web_deploy_path)) | regex_replace('\\/$', '') }}"
+    version: "{{ mumble_web.version }}"
+    checksum: "{{ mumble_web.checksum }}"
+    reinstall: "{{ mumble_web.reinstall | default(false)}}"
+    config: "{{ mumble_web.config | default({}) }}"
+
+- name: Calculate facts for installation
+  set_fact:
+    archive_file: "{{ global_cache_dir }}/mumble-web-{{ version }}.tar.gz"
+    build_dir: "{{ build_base }}/mumble-web-{{ version }}"
+    deploy_base: "{{ deploy_path | dirname }}"
+
+- name: Create deployment parent directory
+  file:
+    path: "{{ deploy_base }}"
```

Comment:
Setting an owner and `0700` permissions is probably better here

Reply:
I think this is usually `0755` and `root:root`, which is the default for `file`.
But I will add those to make it explicit.